src/pages/Lend/FilterAlpha/LendFilterAlpha.vue

```diff
@@ -11,6 +11,41 @@
 			</div>
 			<div class="lg:tw-absolute lg:tw-w-full lg:tw-h-full lg:tw-top-0 lg:tw-pt-8">
 			</div>
+			<kv-page-container>
+				<h2 class="tw-mb-2">
+					Make a loan, change a life
+				</h2>
+				<h3 class="tw-mb-3">
+					Each Kiva loan helps people build a better future for themselves and their families.
+				</h3>
+				<kv-grid class="tw-grid-cols-2 md:tw-grid-cols-3">
+					<div class="tw-bg-gray-300 text-center sm:text-left"><p>Filters</p>
+					</div>
+
+					<div class="tw-col-span-2 tw-bg-white-300">
```

Comment:
Minor, I don't believe the `tw-bg-white-300` class is doing anything and can be removed.